src/quacc/atoms/phonons.py

```diff
@@ -62,7 +62,7 @@ def get_phonopy(
         n_supercells = np.round(np.ceil(min_length / atoms.cell.lengths()))
         supercell_matrix = np.diag([n_supercells, n_supercells, n_supercells])
     else:
-        supercell_matrix = None
```

Comment:
This logic should all be taken care of in #1654.

Reply:
Yes sorry I forgot that this was here, changing back